### PR TITLE
Removed Dolt object refs from SQL context

### PR DESCRIPTION
**Pull request by oscarbatori**

### CODE REVIEW

File: tests/sql/test_sql.py [hunk 1]
```diff
@@ -56,3 +56,10 @@ def test_commit_tables(with_test_tables):
 @pytest.mark.skip()
 def test_commit_tables_empty_commit():
     pass
+
+
+def test_show_tables(with_test_tables):
+    dolt = with_test_tables
+    with DoltSQLServerContext(dolt, TEST_SERVER_CONFIG) as dssc:
+        tables = dssc.tables()
+        assert TEST_TABLE_ONE in tables and TEST_TABLE_TWO in tables
```
Comment (max-hoffman) [inline tests/sql/test_sql.py:65]: run black

**Review by max-hoffman — APPROVED** [review]
👍

File: tests/sql/test_sql.py (hunk 1 above)
Comment (oscarbatori) [reply tests/sql/test_sql.py:65]: Black ignores tests...

Comment (max-hoffman) [reply tests/sql/test_sql.py:65]: ah yeah my bad, missing `\n` doesn't matter as much there

**Pull request merged**